### PR TITLE
change wording

### PR DESCRIPTION
**Pull request by cramja**
Read the whitepaper and made suggested edits. There's still a question in my mind if it's really a whitepaper- reads more like a product overview rather than a technical dive. That's actually fine, I think this needs an overview but then the naming is wrong if that's what it is. Pretty clear outline overall, though, I didn't edit the `Tax Filing` section as I didn't understand what it was trying to say.

@huonghovan

### CODE REVIEW

File: whitepaper.md [hunk 1]
```diff
@@ -52,27 +54,25 @@ A user bought 1 BTC on [Coinbase](https://coinbase.com) at 35K and bought anothe
...
-
-To preserve privacy, CoinTrail chooses not to have a central server for the app to phone home, to not collect any crash logs...This leads to some challenges on distribution and supporting the community.
+CoinTrail does not to have a central server for the app to harvest user information in order to protect user privacy. However, this also means that CoinTrail cannot collect any crash logs. This leads to challenges on app distribution, and supporting the user community.
 
-Tax calculation is not as complex as one might think. It can be done manually with a small number of transactions but it's gonna be a challenge with larger transactions over longer periods of time. Many SaaS tax services take advantage and charge users with a high fee, recurrence subscription...
+Tax calculation is not as complex as one might think. It can be done manually with a small number of transactions but it's a challenge with many transactions over long periods of time. Many SaaS tax services take advantage and charge users with a high fee, and recurrencing subscriptions. CoinTrail takes an inexpensive yet powerful, local-first approach instead.
```
Comment (huonghovan) [inline whitepaper.md]: `privacy-first` instead of `local-first`?

**Review by huonghovan — APPROVED** [review]
Looks good, yeah will evolve into more technical white-paper.

**Comment by huonghovan** [comment]
let note down the questions can discuss here or on different PR.

**Comment by huonghovan** [comment]
Could u take a crack at appstore description as well? https://apps.apple.com/us/app/cointrail/id1614887276

**Comment by cramja** [comment]
App store, just cleaned up a little

```
CoinTrail is the best crypto tax calculator for HODLing without the fear of tax burden. CoinTrail instantly provides information about your taxable crypto income, realized gain/loss, and breaks down tax-lots in detail for your next cryptocurrency sell.


- Secure and Decentralized

Only you have access to your data. All imported transactions and settings are kept securely on your device. No sign up needed.


- All In One Place

CoinTrail's thoughtful interface allows you to easily review all your holdings, transactions, cost basis, fees, and taxable income in one place.


- Smart Transactions Grouping

CoinTrail's unique and effective algorithm detects duplicate transactions, matches transactions, and tracks cost basis across exchanges.


- Intuitive Import Wizard

CoinTrail supports automatic import for popular exchanges and for all others, you can manually import CSV files exported from your exchange or wallet.


Importing transactions from crypto exchanges and wallets:
- Coinbase
- CoinbasePro
- Cashapp
- CelsiusNetwork
- Kraken
- ElectrumWallet
- ExodusWallet
- Custom CSV format


Syncing transactions from crypto exchanges:
- Coinbase
- CoinbasePro
- CelsiusNetwork
- Kraken
- Gemini
- Ftxus


Syncing transactions from mining pools:
- F2Pool
- ViaBTC


Exporting capital gain/loss tax form:
- Standard 8949
- TurboTax
```

**Comment by cramja** [comment]
2 basic questions

For the app store description:

> detects duplicate transactions, matches transactions, and tracks cost basis across exchanges.

How is matching txn different than tracking cost basis cross exchange?

From the whitepaper

>  although the default Genesis portfolio should be enough for 95% of the use cases. 

maybe best to leave out "Genesis" since the name for the default doesn't add anything, save a biblical flavor.

Overall the need for different portfolios based on filing situation probably could use a blog post or something for a non HODL like myself to understand why that use case matters.

Maybe for an offline discussion - who's found traction with this app? Also, is there value in the unifying-many-wallets feature and is that commensurate with the value of the tax feature? e.g. would people use it just for that alone?

**Comment by huonghovan** [comment]
Yeah, I think let do 1 or 2 more sessions on the use cases then you good to go. Are u able to merge the PR or I have todo it?

**Comment by cramja** [comment]
SG - you'll need to merge as I don't have the option

**Comment by huonghovan** [comment]
> SG - you'll need to merge as I don't have the option

hmm, could u try again? it might be something todo with team permission.

**Pull request merged**